### ISSUE TITLE
addImageBlobHook inserts image twice (once as base64 and custom)

### ISSUE THREAD

**Issue by jalyna** [nhn/tui.editor#1588]
## Describe the bug
Thanks for the great work on the latest upgrade!

The `addImageBlobHook` does not work anymore as expected since upgrading to 3.0.0. I couldn't find anything about it in the migration guide so I'm not sure if I'm missing something here.

## To Reproduce
Set the options like this:

```js
hooks: {
  addImageBlobHook: (fileOrBlob, callback) => {
    callback("https://example.com/picture.png");
  },
},
```

Click on "Image" and select a file to upload and click on "OK"

Then you will see once the custom image url (`"https://example.com/picture.png"`) and once the image as base64:

## Expected behavior
Only the custom image is added.

## Screenshots
![Screenshot 2021-06-18 at 11 48 31](https://user-images.githubusercontent.com/5362483/122542847-6fa22300-d02b-11eb-9079-76d51d3b0ab2.png)

## Desktop
 - OS: MacOS
 - Browser: Chrome
 - Version: 90.0.4430.93 

## Additional context
Introduced with 3.0.0

**Comment by intermyr** [comment]
Having the same issue, only base64 image comes before custom one

**Comment by z-950** [comment]
``` js 
editor.removeHook('addImageBlobHook'); // remove the default hook
editor.addHook('addImageBlobHook', (blob, callback) => { ... });
```
You can use this to solve the problem, instead of using option.

**Comment by js87zz** [comment]
@jalyna 
Sorry for late replying. I will fix it.
Thanks!

**Comment by js87zz** [comment]
@jalyna @intermyr 
The bug has been fixed and released in [v3.0.1](https://github.com/nhn/tui.editor/releases/tag/editor%403.0.1).
